test_wazuh/test_fim/test_ambiguous_confs/test_ambiguous_simple.py

```diff
@@ -119,7 +145,7 @@ def report_changes_validator(event):
             diff_file = os.path.join(WAZUH_PATH, 'queue', 'diff', 'local')
             if sys.platform == 'win32':
                 diff_file = os.path.join(diff_file, 'c')
-                diff_file = os.path.join(diff_file, folder.strip('C:\\'), file)
+                diff_file = os.path.join(diff_file, folder.strip('c:\\'), file)
             else:
                 diff_file = os.path.join(diff_file, folder.strip('/'), file)
```

Comment:
We can join lines 148,149 and 150 and remove the else in a single line using PREFIX such as:

diff_file = os.path.join(diff_file, folder.strip(PREFIX), file)